src/lib/y2network/sysconfig/hostname_reader.rb

```diff
@@ -95,11 +95,13 @@ def hostname_from_install_inf
         host.empty? ? nil : host
       end
 
-      # Reads the hostname known to the resolver
+      # Reads the (transient) hostname known to the resolver
       #
-      # @return [String] Hostname
+      # @return [String, nil] Hostname or nil if transient hostname is not known
       def hostname_from_resolver
         Yast::Execute.on_target!("/usr/bin/hostname", "--fqdn", stdout: :capture).strip
+      rescue Cheetah::ExecutionFailed
```

Comment:
just not sure, is it also logged in case that command failed? I think yes, but not sure.

Reply:
yes it is ... by cheetah